### PR TITLE
Fixing issues adding proper docker support

### PR DESCRIPTION
**Pull request by kthakar1990**
-  Added Dockerfile for Front End
- Updated Dockerfile and Main.py to make it work on local without the need for swagger
- Added some UI changes in form
- WIP for docker-compose.. so we can run both docker images as one container - will add in separate PR

### CODE REVIEW

**Comment by kadler** [comment]
Why are you replacing the optimized gunicorn Docker setup with the built-in Flask server?

**Comment by kthakar1990** [comment]
> Why are you replacing the optimized gunicorn Docker setup with the built-in Flask server?

The other image was not working properly on local host.. I tried to build image and run container but wasn't able to access it... so I had to make this change to make it work locally but I will try to see if I can go back to that efficient image because I do understand the performance reasons...

**Comment by kadler** [comment]
I had no issues running it locally. What were you running in to?

**Comment by kthakar1990** [comment]
> I had no issues running it locally. What were you running in to?

I am doing docker run command.. `docker run --rm -d -p 8080:8080 --name backend <image name>` and it is not working on browser.. gives bad gateway and with some changes the docker won't even start..

**Comment by kadler** [comment]
You need to forward to port 80 in the container, not 8080 `docker run --rm -d -p 8080:80 --name backend <image name>`

File: python/app/app/main.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 import pickle
 import pandas as pd
 from flask import Flask, jsonify, request
+from flask_cors import CORS
```
Comment (kadler) [inline python/app/app/main.py:7]: This needs to be added to pyproject.toml, then run `poetry lock`, and `poetry export -f requirements.txt > requirements.txt`

File: python/app/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# install dependencies
+RUN pip install --no-cache-dir --upgrade pip
+
+# copy requirements.txt
+COPY ./requirements.txt /app/requirements.txt
+
+# install project requirements
+RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install -U flask-cors
```
Comment (kadler) [inline python/app/Dockerfile]:
```suggestion
```
This should be in the pyproject.toml

Comment (kthakar1990) [reply python/app/Dockerfile]: done!

**Review by kadler — APPROVED** [review]
LGTM

**Pull request merged**